app/Http/Controllers/Kata7/StringEndController.php

```diff
@@ -29,14 +51,18 @@ public function __construct(StringEndService $stringEndService)
      */
     public function stringEndFormSubmit(StringEndRequest $request): RedirectResponse
     {
-        $stringValue = $request->input('text');
-        $stringEnd = $request->input('textEnding');
-        $messageString = $this->stringEndService->checkIfStringMatchesTheGivenEnding($stringValue, $stringEnd);
+        $stringEndDetails = $request->only([
+            'text',
+            'text_ending'
+        ]);
+
+        $this->store($stringEndDetails);
+        $messageString = $this->stringEndService->checkIfStringMatchesTheGivenEnding($stringEndDetails['text'], $stringEndDetails['text_ending']);
```

Comment:
I know its not added in this pull request, but the based on the function name "checkIfStringMatches" I would expect it to return a boolean, not a string. If someone else where to use it like a boolean, it would always be true.

Reply:
Good point. I have renamed the method to 'checkIfStringMatchesTheGivenEndingAndReturnStringMessage'

Reply:
As discussed the service has been removed and the 'string end check' is done in the controller.